content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/sboms-and-attestations.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "SBOMs and Attestations in Chainguard Enforce"
+type: "article"
+description: "An overview of how Chainguard Enforce handles attestations and SBOMs"
+date: 2023-03-13T15:56:52-07:00
+lastmod: 2023-03-13T15:56:52-07:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 600
+toc: true
+---
+
+In order to fully leverage the capabilities offered by Chainguard Enforce, it's important to establish a process for creating quality *Software Bills of Materials* (SBOMs). An SBOM is a formal record that contains the details and supply chain relationships (such as dependencies) of the components used in building given software.
```

Comment:
I think linking out to our SBOM resources would be helpful. Also "given software" sounds a little odd to me. If you want to keep it, i would rephrase it as "a given software artifact"

```suggestion
In order to fully leverage the capabilities offered by Chainguard Enforce, it's important to establish a process for creating quality *Software Bills of Materials* (SBOMs). An [SBOM](/open-source/sbom/) is a formal record that contains the details and supply chain relationships (such as dependencies) of the components used in building software.
```